### ISSUE TITLE
Trash files in working directory after setup

### ISSUE THREAD

**Issue by ad-m** [shivammathur/setup-php#65]
**Describe the bug**

When launched, my working directory contains changes that are not expected or expected. Some files were not cleaned properly after setup scripts were executed.

**To Reproduce**
Please provide the GitHub Action `.yml` file.
```yml
name: CI

on:
  schedule:
  - cron: 0 0,8,16 * * 0-5

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@master
      - name: Setup PHP
        uses: shivammathur/setup-php@master
        with:
          php-version: '5.6'
      - name: Verify working-dir clean
        run: git status
```

**Expected behavior**

Working directory have following unstaged files:
* 5.6linux.sh
* phalcon.sh

**Screenshots**

https://github.com/ad-m/cbosa/commit/e73952c05c361050e7006609b1120b96aa19f5ae/checks?check_suite_id=284329170

**Additional context**

**Comment by shivammathur** [comment]
@ad-m Thanks for reporting this.
Will be fixed in the next release `1.4.5`.

**Comment by shivammathur** [comment]
@ad-m Fixed in #66
